Tools/autotest/carbonix.py

```diff
@@ -76,9 +78,9 @@ def TestESCTelemetry(index):
 
             # Check that the prearm disable parameter works
             self.progress('Checking prearm disable parameter')
-            self.set_parameter('BIT_PREARM_DIS', 0b1)
+            self.set_parameter('BIT_PREARM_DIS', 0b101)
             self.wait_ready_to_arm()
-            self.set_parameter('BIT_PREARM_DIS', 0)
+            self.set_parameter('BIT_PREARM_DIS', 0b100)
```

Comment:
Here, we should use a get-set-restore pattern (combined with the previous suggestion of setting `0b100` in the top-level) instead of directly setting the values before and after.

This will make it easier if we ever need make more things default to disabled.